README.md

```diff
@@ -1,2 +1,108 @@
-# terraform-inspec-template
-A template repository for terraform-inspect projects
+# terraform-kitchen-template
+A template repository for terraform projects with kitchen terraform tests, pre-commit hooks, and GitHub actions for CI, PR labeler and Relase Drafter. See [Development](#development) section for details.
+
+# update-badge
+**update badge link below!**
+
+![kitchen-tests](https://github.com/BrownUniversity/terraform-kitchen-template/workflows/kitchen-tests/badge.svg)
+
+
+# Contents:
+
+- [Getting Started](#getting-started)
+- [How to use this module](#how-to-use-this-module)
+- [Requirements](#requirements)
+- [Providers](#providers)
+- [Inputs](#inputs)
+- [Testing](#testing)
+- [Development](#development)
+
+
+## Getting Started
+
+This module depends on you having credentials of some kind. The module looks for a credential file in JSON format. For GCP, you should export the following:
+
+```
+GOOGLE_APPLICATION_CREDENTIALS=/path/to/file.json
+```
+
+## How to use this module
+
+This repository defines a [Terraform module](https://www.terraform.io/docs/modules/usage.html), which you can use in your
+code by adding a `module` configuration and setting its `source` parameter to URL of this repository. See the [examples](/examples) folder for guidance
```

Comment:
This text should probably include the workflow for copying the template and customizing it, right?

Reply:
That's a great point!